native/effektio/src/platform/desktop.rs

```diff
@@ -18,17 +23,102 @@ pub async fn new_client_config(base_path: String, home: String) -> Result<Client
     Ok(builder)
 }
 
-pub fn init_logging(filter: Option<String>) -> Result<()> {
+static mut FILE_LOGGER: Option<Arc<fern::ImplDispatch>> = None;
```

Comment:
same as above, please use a lazy-static and/or mutex here for a global not `unsafe`.

Reply:
Yeah, I fixed that by `lazy_static`.
Please reference https://github.com/acterglobal/a3/pull/488/commits/91869ec5f90d3912b76ec5034ff404afb4b42127